TicTacToe2/TicTacToe/Board.cs

```diff
@@ -70,6 +70,45 @@ public IEnumerable<char[]> GetDimensions()
 
             // fill dimensions with data from spaces
 
+            // horizontal
+            char[] x = new char[Size];
+            for (int row = 0; row < Size; row++)
+            {
+
+                for (int col = 0; col < Size; col++)
+                {
+                    x[row] = spaces[row, col];
```

Comment:
I strongly recommend using variable names x and y to iterate over a 2d array. It's easy to get confused about the order of the indexes when using row and column (should be spaces[col, row]) Also, the variable you're using to store the array that represents the dimension should have a more descriptive name and should be declared closer in scope to where it's used.

```c#
for (int y = 0; y < Size; y++) // Iterate over y values in the spaces array. This results in looking at one row per iteration
{
    var row = new char[Size]; // This array represents a row of spaces. The outer loop is being used to determine which row we're looking at, so this is the appropriate place to declare this value
    for (int x = 0; x < Size; x++) // Iterate over x values in the spaces array. The outer loop is determining which row we're working with. The inner loop is looking at each value in that row
    {
        row[x] = spaces[x, y]; // Add each value in row y of the spaces array to the row array
    }
    dimensions.Add(row); // Add the row array to the dimensions list
}
```